core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -34,10 +34,15 @@ oppia.directive('learnerDashboardIcons', [
         'LearnerDashboardActivityIdsObjectFactory',
         'LearnerPlaylistService',
         function(
-            $scope, LearnerDashboardIdsBackendApiService,
-            LearnerDashboardActivityIdsObjectFactory,
-            LearnerPlaylistService) {
+          $scope, LearnerDashboardIdsBackendApiService,
+          LearnerDashboardActivityIdsObjectFactory,
+          LearnerPlaylistService) {
           $scope.activityIsCurrentlyHoveredOver = true;
+          $scope.playlistTooltipIsEnabled = false;
+
+          $scope.changePlaylistTooltipState = function(playlistTooltipState) {
```

Comment:
Let's still go with two separate functions: enablePlaylistTooltip() and disablePlaylistTooltip(). We try to avoid using the word state in Oppia because it's already a term of art (it refers to the cards that make up an exploration).

Reply:
Done :+1: